internal/cmd/login.go

```diff
@@ -523,8 +523,7 @@ func deviceFlowLogin(client *api.Client, cli *CLI) (*api.LoginResponse, error) {
 	defer spinner.Stop()
 
 	var spinnerCount int = 0
-	var pollResp *api.DevicePollResponse
-loop:
```

Comment:
I was able to remove both the loop label and the var by returning directly from the `case` instead of breaking from the case and returning at the bottom. I think this makes the function easier to read because you don't have to jump from the break to the return.